src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -472,6 +473,49 @@ void GeneralPage::initLayout(SettingsLayout &layout)
             return fuzzyToInt(args.value, 0);
         });
 
+    layout.addTitle("Streamer Mode");
+    layout.addDescription(
+        "Chatterino can automatically change behavior if it "
+        "detects that \"OBS Studio\" is running.\nSelect which "
+        "things you want to change while streaming");
+
+    layout.addDropdown<std::underlying_type<StreamerModeSetting>::type>(
+        "Enable Streamer Mode", {"Yes", "No", "Detect OBS (win)"},
+        s.enableStreamerMode,
+        [](int val) {
+            switch (val)
+            {
+                case StreamerModeSetting::Enabled:
+                    return "Yes";
+                case StreamerModeSetting::Disabled:
+                    return "No";
+                case StreamerModeSetting::DetectObs:
+                    return "Detect OBS (win)";
+                default:
+                    return "";
+            }
+        },
+        [](DropdownArgs args) {
+            if (args.value == "Yes")
+            {
+                return StreamerModeSetting::Enabled;
+            }
+            else if (args.value == "No")
+            {
+                return StreamerModeSetting::Disabled;
+            }
+            else
+            {
+                return StreamerModeSetting::DetectObs;
+            }
+        },
```

Comment:
Wondering why this part is so compicated.
```suggestion
        s.enableStreamerMode, [](int value) { return value; },
        [](DropdownArgs args) {
            return static_cast<StreamerModeSetting>(args.index);
        },
```